### PR TITLE
Branding changed to Studio DRM - also updated the player to the latest version.

### PR DESCRIPTION
**Pull request by bengo1990**
Appropriate TODOs placed within the code for phase 2 of this PR (when doc locations get finalised, etc).

package.json file will also need amending in the future... can't put a TODO in the file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-    "name": "vuplay-jwplayer",
+    "name": "studiodrm-jwplayer",
     "version": "1.0.0",
-    "description": "A basic implementation of the JW Player with the option to integrate VUDRM",
+    "description": "A basic implementation of the JW Player with the option to integrate StudioDRM",
```
Comment (PerritonBranch) [inline package.json:4]:
```suggestion
    "description": "A basic implementation of the JW Player with the option to integrate Studio DRM",
```

File: README.md [hunk 2]
```diff
@@ -5,52 +5,63 @@
 
 ## Description
 
-This repository will demostrate how to use [VUDRM](https://vudrm.vualto.com/) with the [JW Player](https://jwplayer.com).
-If you have any questions please contact <support@vualto.com>
+#### !!!!TODO!!!!: Change the below to the JW Studio DRM docs
```
Comment (PerritonBranch) [inline README.md:8]: Why is this TODO being shouted?

File: README.md [hunk 3]
```diff
@@ -5,52 +5,63 @@
 
 ## Description
 
-This repository will demostrate how to use [VUDRM](https://vudrm.vualto.com/) with the [JW Player](https://jwplayer.com).
-If you have any questions please contact <support@vualto.com>
+#### !!!!TODO!!!!: Change the below to the JW Studio DRM docs 
+This repository will demonstrate how to use [Studio DRM](https://jw-studio-drm-docs) with [JW Player](https://jwplayer.com).
+If you have any questions please contact <support@jwplayer.com>
```
Comment (PerritonBranch) [inline README.md:10]: Why are you surrounding this with `<>`?
Are trying to achieve this?
```suggestion
If you have any questions please contact [support@jwplayer.com](support@jwplayer.com).

```

File: README.md [hunk 4]
```diff
@@ -5,52 +5,63 @@
...
+Retrieve the [JW Player key](https://dashboard.jwplayer.com/#/players/downloads) and copy the license key for the appropriate player in the `Downloads` section.
 
 In order to allow DRM encrypted playback in chrome (<https://goo.gl/EEhZqT>), SSL has been enabled for the demo. You will get a warning about an invalid cert `NET::ERR_CERT_AUTHORITY_INVALID` but this can safely be ignored.
 
 ### Browser support
 
 The browser must support [encrypted media extensions](https://www.w3.org/TR/2016/CR-encrypted-media-20160705/).
 Currently this includes the latest versions of Chrome, Firefox, Internet Explorer 11 and Edge.
-For a complete breakdown of supported media extensions please contact support@vualto.com
+For a complete breakdown of supported media extensions please contact <support@jwplayer.com>
```
Comment (PerritonBranch) [inline README.md:48]: As above.

File: README.md [hunk 5]
```diff
@@ -5,52 +5,63 @@
...
 
 ## Useful links
 
-### VUDRM
+### StudioDRM
```
Comment (PerritonBranch) [inline README.md:52]:
```suggestion
### Studio DRM

```

File: README.md [hunk 6]
```diff
@@ -5,52 +5,63 @@
...
 
--   [Contact vualto](https://www.vualto.com/contact-us/)
--   [VUDRM](https://vudrm.vualto.com/)
--   [VUDRM token documentation](https://docs.vualto.com/projects/vudrm/en/latest/VUDRM-token.html)
+-   [Contact JW Player](https://support.jwplayer.com/)
+#### !!!!TODO!!!!: Change the below to the JW Studio DRM docs
+-   [StudioDRM](https://jw-studio-drm-docs)
```
Comment (PerritonBranch) [inline README.md:56]:
```suggestion
-   [Studio DRM](https://jw-studio-drm-docs)

```

File: README.md [hunk 7]
```diff
@@ -5,52 +5,63 @@
...
+-   [StudioDRM token documentation](https://jw-studio-drm-token-docs)
```
Comment (PerritonBranch) [inline README.md:57]:
```suggestion
-   [Studio DRM token documentation](https://jw-studio-drm-token-docs)

```

File: index.html [hunk 8]
```diff
@@ -3,24 +3,25 @@
...
-            #vuplay-container {
+            #studiodrm-container {
                 width: 50%;
             }
         </style>
     </head>
     <body>
-        <div id="vuplay-container"></div>
+        <div id="studiodrm-container"></div>
+        <!--TODO: For the JW Player src - look at using the version hosted on JW Player's CDN-->
```
Comment (PerritonBranch) [inline index.html:17]: Can we stick to writing the TODO's one way. I prefer this over the shouty one as iirc this is how we do it everywhere else.

**Review by PerritonBranch — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (bengo1990) [reply README.md:52]: Done!

File: README.md (hunk 7 above)
Comment (bengo1990) [reply README.md:57]: Done!

File: README.md (hunk 2 above)
Comment (bengo1990) [reply README.md:8]: I honestly don't know! Changed!

File: README.md (hunk 3 above)
Comment (bengo1990) [reply README.md:10]: The <> does the same thing as your suggested change.

File: index.html (hunk 8 above)
Comment (bengo1990) [reply index.html:17]: Yes. It was a moment of pure madness!

File: README.md [hunk 9]
```diff
@@ -5,52 +5,63 @@
...
-2. In the file `vuplay-jwplayer.js` replace `<dash-stream-url>` with your [MPEG-DASH](https://en.wikipedia.org/wiki/Dynamic_Adaptive_Streaming_over_HTTP) stream URL.
-3. In the file `vuplay-jwplayer.js` replace `<hls-stream-url>` with your [HLS](https://developer.apple.com/streaming/) stream URL.
-4. In the file `vuplay-jwplayer.js` replace `<fairplay-cert-url>` with the URL to the fairplay certificate. Please contact support@vualto.com if you do not have this URL.
-5. In the file `vuplay-jwplayer.js` replace `<your-vudrm-token>` with a VUDRM token from [https://admin.vudrm.tech](https://admin.vudrm.tech)
+2. In the file `studiodrm-jwplayer.js` replace `<dash-stream-url>` with your [MPEG-DASH](https://en.wikipedia.org/wiki/Dynamic_Adaptive_Streaming_over_HTTP) stream URL.
+3. In the file `studiodrm-jwplayer.js` replace `<hls-stream-url>` with your [HLS](https://developer.apple.com/streaming/) stream URL.
+4. In the file `studiodrm-jwplayer.js` replace `<fairplay-cert-url>` with the URL to the fairplay certificate. Please contact <support@jwplayer.com> if you do not have this URL.
+#### TODO: Change the below (no.5) to the JW Studio DRM docs
```
Comment (somerobmitchell) [inline README.md:31]: TODO needs link to Jira story

File: README.md [hunk 10]
```diff
@@ -5,52 +5,63 @@
...
 
 ## Instructions
 
 ### Install dependencies
 
 1. Install [npm](https://www.npmjs.com/)
 2. Install the [grunt-cli](https://www.npmjs.com/package/grunt-cli): `npm install -g grunt-cli`
-3. Clone the repository: `git clone git@github.com:Vualto/vuplay-jwplayer.git`
-4. Navigate to the project's root folder: `cd vuplay-jwplayer`
+#### TODO: Change the below (no.3) to the JW GitHub repo structure name
```
Comment (somerobmitchell) [inline README.md:20]: TODO needs link to Jira story

File: README.md [hunk 11]
```diff
@@ -5,52 +5,63 @@
 
 ## Description
 
-This repository will demostrate how to use [VUDRM](https://vudrm.vualto.com/) with the [JW Player](https://jwplayer.com).
-If you have any questions please contact <support@vualto.com>
+#### TODO: Change the below to the JW Studio DRM docs
```
Comment (somerobmitchell) [inline README.md:8]: TODO needs link to Jira story

File: README.md [hunk 12]
```diff
@@ -5,52 +5,63 @@
...
+5. In the file `studiodrm-jwplayer.js` replace `<your-studiodrm-token>` with a Studio DRM token from [https://jw-studio-drm-token-docs](studiodrm-token-docs)
 6. In the file `index.html` replace `<jwplayer-key>` with your JWPlayer key.
 7. Run `npm run build` in the project's root. This will create a `dist` folder that contains all the files needed to run this demo. 
-8. Add the host `jwplayer.vuplay.local.drm.technology` to your local machine's hosts file.
-9. Load a supported browser and go to `https://jwplayer.vuplay.local.drm.technology:14703`
+8. Add the host `jwplayer.studiodrm.local` to your local machine's hosts file.
+9. Load a supported browser and go to `https://jwplayer.studiodrm.local:14703`
 
-You will need an enterprise license with JW Player in order to playback DRM encrypted content. Please contact support@vualto.com or [JW Player](https://www.jwplayer.com/pricing/) for more information. 
+You will need an enterprise license with JW Player in order to playback DRM encrypted content. Please contact <support@jwplayer.com> or [JW Player](https://www.jwplayer.com/pricing/) for more information.
```
Comment (somerobmitchell) [inline README.md:38]:
```suggestion
You will need an enterprise license with JW Player in order to playback DRM encrypted content. Please contact <support@jwplayer.com> or see [JW Player pricing](https://www.jwplayer.com/pricing/) for more information. 

```

File: README.md [hunk 13]
```diff
@@ -5,52 +5,63 @@
...
+Retrieve the [JW Player key](https://dashboard.jwplayer.com/#/players/downloads) and copy the license key for the appropriate player in the `Downloads` section.
 
 In order to allow DRM encrypted playback in chrome (<https://goo.gl/EEhZqT>), SSL has been enabled for the demo. You will get a warning about an invalid cert `NET::ERR_CERT_AUTHORITY_INVALID` but this can safely be ignored.
 
 ### Browser support
 
 The browser must support [encrypted media extensions](https://www.w3.org/TR/2016/CR-encrypted-media-20160705/).
 Currently this includes the latest versions of Chrome, Firefox, Internet Explorer 11 and Edge.
-For a complete breakdown of supported media extensions please contact support@vualto.com
+For a complete breakdown of supported media extensions please contact <support@jwplayer.com>
```
Comment (somerobmitchell) [inline README.md:48]: Is there not a doc page that we can link to for this? I imagine it is documented in the public docs.

File: README.md [hunk 14]
```diff
@@ -5,52 +5,63 @@
...
 ## Useful links
 
-### VUDRM
+### Studio DRM
 
--   [Contact vualto](https://www.vualto.com/contact-us/)
--   [VUDRM](https://vudrm.vualto.com/)
--   [VUDRM token documentation](https://docs.vualto.com/projects/vudrm/en/latest/VUDRM-token.html)
+-   [Contact JW Player](https://support.jwplayer.com/)
+#### !!!!TODO!!!!: Change the below to the JW Studio DRM docs
```
Comment (somerobmitchell) [inline README.md:55]: TODOs need the jira link

File: studiodrm-jwplayer.js [hunk 15]
```diff
@@ -7,11 +7,13 @@
...
 
-    // Please login to https://admin.vudrm.tech to generate a VUDRM token.
-    var vudrmToken = "<your-vudrm-token>";
+    // TODO: Change the below to the JW Studio DRM docs 
+    // Please login to https://admin.vudrm.tech to generate a StudioDRM token.
+    var studioDrmToken = "<your-studiodrm-token>";
     
-    // setup jwplayer, passing the stream URLs and DRM configurations.  
-    jwplayer("vuplay-container").setup({
+    // TODO: Change lines 23, 30, 50 with the new header name... X-STUDIODRM-TOKEN?
```
Comment (somerobmitchell) [inline studiodrm-jwplayer.js:14]: TODOs need the jira link

**Review by somerobmitchell — CHANGES_REQUESTED** [review]

File: README.md (hunk 11 above)
Comment (bengo1990) [reply README.md:8]: Actioned.

File: README.md (hunk 10 above)
Comment (bengo1990) [reply README.md:20]: Actioned.

File: README.md (hunk 9 above)
Comment (bengo1990) [reply README.md:31]: Actioned.

File: README.md (hunk 14 above)
Comment (bengo1990) [reply README.md:55]: Actioned.

File: README.md (hunk 12 above)
Comment (bengo1990) [reply README.md]: Actioned.

**Comment by bengo1990** [comment]
@PerritonBranch @somerobmitchell @askolodny Spoke to Kevin - he has asked to wait for him to finish the Token v2 JW docs prior to this being merged in. Please review the document, if there are no changes, I will merge when he is happy.

**Review by somerobmitchell — APPROVED** [review]

File: README.md [hunk 16]
```diff
@@ -5,52 +5,59 @@
...
+-   [Contact JW Player](https://support.jwplayer.com/)
+-   [Studio DRM](https://developer.jwplayer.com/jwplayer/docs/studio-drm-standalone-getting-started)
+-   [Studio DRM token documentation](https://developer.jwplayer.com/jwplayer/docs/studio-drm-token-v2)
+
+### JW Player
+
+-   [Homepage](https://www.jwplayer.com/)
+-   [Pricing](https://www.jwplayer.com/pricing/)
+-   [Developer Portal](https://developer.jwplayer.com)
+-   [JW Player documentation on the configuration of DRM with JWPlayer](https://developer.jwplayer.com/jw-player/docs/developer-guide/customization/configuration-reference/#drm)
```
Comment (somerobmitchell) [inline README.md:60]: should this link be this one: https://developer.jwplayer.com/jwplayer/docs/studio-drm-standalone-web-players
as this is for standalone?

**Review by somerobmitchell — CHANGES_REQUESTED** [review]

File: README.md (hunk 16 above)
Comment (somerobmitchell) [reply README.md:60]: or even here: https://developer.jwplayer.com/jwplayer/docs/studio-drm-standalone-web-players#jw-player

Comment (bengo1990) [reply README.md]: Actioned.

**Review by PerritonBranch — DISMISSED** [review]

**Review by PerritonBranch — APPROVED** [review]

**Review by somerobmitchell — APPROVED** [review]

**Pull request merged**